### ISSUE TITLE
Template variables could not be initialized: e.replace is not a function

### ISSUE THREAD

**Issue by lysyy** [akpw/mktxp#6]
Template variables could not be initialized: e.replace is not a function -  with Grafana dashboard. id=13679

**Comment by akpw** [comment]
hi @lysyy, a few quick questions:
 - which grafana version do you have? 
 - do you see this in a specific panel within the dashboard? Which one?
 - ideally, can you attach the query response where you got this error?

**Comment by akpw** [comment]
closing due to lack of activity

**Comment by djieno** [comment]
I've got this error too in Grafana v7.5.11 (6f8c1d9fe4)

![image](https://github.com/akpw/mktxp/assets/59441232/5e3ab202-0eaf-4399-a81d-e5fc9d5b7e1b)

in prometheus mktxp_system_identity_info outputs fine :
`mktxp_system_identity_info{endpoint="mktxp-exporter", instance="10.masked.600:49090", job="mktxp-metrics", name="mtik-mainrouterA", namespace="cattle-monitoring-system", routerboard_address="10.masked.1", routerboard_name="main", service="mktxp-metrics"}`

**Comment by djieno** [comment]
I solved it by changing data source inputs

This is done by pressing the settings 'wheel' in the top right corner and then apply the correct data source. I think the dashboard could use 'default' (which will pickup the default / prometheus)  instead. 

Example for the system_identity
![image](https://github.com/akpw/mktxp/assets/59441232/8bd81322-941e-45a2-a149-4dfb5f63a6c8)

I'm sure there is a simpler way of correcting the data source in one go by changing it into the yaml. 
![image](https://github.com/akpw/mktxp/assets/59441232/d06c6cdf-6ecd-425e-b253-4ef75ea631d6)

For now - Happy I got some nice dashboard...thanks!
